ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/OVERLAY_ROUTING_PROTOCOL_HER_L3LEAF3.cfg

```diff
@@ -0,0 +1,123 @@
+!RANCID-CONTENT-TYPE: arista
+!
+vlan internal order ascending range 1006 1199
+!
+transceiver qsfp default-mode 4x10G
+!
+service routing protocols model multi-agent
+!
+hostname OVERLAY_ROUTING_PROTOCOL_HER_L3LEAF3
+!
+no enable password
+no aaa root
+!
+vlan 2900
+   name L2VLAN_ON_ALL_LEAFS
+!
+vlan 2903
+   name L2VLAN_ON_LEAF2-3
+!
+vlan 3900
+   name SVI_ON_ALL_LEAFS
+!
+vlan 3903
+   name SVI_ON_LEAF2-3
+!
+vrf instance MGMT
+!
+vrf instance TEST
+!
+interface Loopback0
+   description EVPN_Overlay_Peering
+   no shutdown
+   ip address 192.168.254.3/32
+!
+interface Loopback1
+   description VTEP_VXLAN_Tunnel_Source
+   no shutdown
+   ip address 192.168.253.3/32
+!
+interface Vlan3900
+   description SVI_ON_ALL_LEAFS
+   shutdown
+   vrf TEST
+!
+interface Vlan3903
+   description SVI_ON_LEAF2-3
+   shutdown
+   vrf TEST
+!
+interface Vxlan1
+   description OVERLAY_ROUTING_PROTOCOL_HER_L3LEAF3_VTEP
+   vxlan source-interface Loopback1
+   vxlan udp-port 4789
+   vxlan vlan 2900 vni 12900
+   vxlan vlan 2903 vni 12903
+   vxlan vlan 3900 vni 13900
+   vxlan vlan 3903 vni 13903
+   vxlan flood vtep 192.168.249.9 192.168.253.1 192.168.253.2 192.168.254.1 192.168.254.2 192.168.254.3 192.168.254.4 192.168.254.5 192.168.254.6 192.168.254.7 192.168.254.9 192.168.254.10 192.168.254.11 192.168.254.12 192.168.254.14 192.168.254.15 192.168.254.16 192.168.254.17 192.168.254.18 192.168.254.21 192.168.254.33 192.168.254.34 192.168.254.35 192.168.254.36 192.168.254.109 192.168.254.111
+!
+ip virtual-router mac-address 00:1c:73:00:dc:01
+!
+ip routing
+no ip routing vrf MGMT
+ip routing vrf TEST
+!
+ip prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+   seq 10 permit 192.168.254.0/24 eq 32
+   seq 20 permit 192.168.253.0/24 eq 32
+!
+ip route vrf MGMT 0.0.0.0/0
+!
+route-map RM-CONN-2-BGP permit 10
+   match ip address prefix-list PL-LOOPBACKS-EVPN-OVERLAY
+!
+router bfd
+   multihop interval 300 min-rx 300 multiplier 3
+!
+router bgp 65003
+   router-id 192.168.254.3
+   maximum-paths 4 ecmp 4
+   neighbor IPv4-UNDERLAY-PEERS peer group
+   neighbor IPv4-UNDERLAY-PEERS send-community
+   neighbor IPv4-UNDERLAY-PEERS maximum-routes 12000
+   redistribute connected route-map RM-CONN-2-BGP
+   !
+   vlan 2900
```

Comment:
vlan statement under router bgp should be removed if we use overlay_routing_protocol different of ebgp or ibgp